woocommerce/woocommerce-ios
# Select default value for paper size in Shipping Label print screen

Fixes #5014 

On Print Shipping Label screen, no paper size is selected by default and the Print button is disabled. There is no emphasis that the paper size should be selected for the Print button to be enabled, so this is not good in terms of discoverability. In this PR I fixed this behavior, selecting a default value for paper size. This also matches with the behavior on Android.

## Testing
1. Make sure your test store has WCShip plugin installed and set up packages for shipping in WPAdmin > WooCommerce > Settings > Shipping.
2. Select an order eligible for creating shipping label and select Create Shipping Label.
3. Proceed through the form to purchase a label.
4. After purchase succeeds, notice that you've navigated to Print Shipping Label screen, and the Print button is enabled with a paper size selected.

<img src="https://user-images.githubusercontent.com/495617/134344335-a2f7d664-2396-4d74-88e2-4a4255eca189.png" width=300 />

Update release notes:
